src/Forms/Compare.cs

```diff
@@ -840,22 +840,9 @@ private void ShowTip(Control control)
             }
         }
 
-        private void subtitleListView1_DragEnter(object sender, DragEventArgs e)
+        private void subtitleListViewDragEnter(object sender, DragEventArgs e)
         {
-            VerifyDragEnter(e);
-        }
-
-        private void subtitleListView2_DragEnter(object sender, DragEventArgs e)
-        {
-            VerifyDragEnter(e);
-        }
-
-        private static void VerifyDragEnter(DragEventArgs e)
-        {
-            if (e.Data.GetDataPresent(DataFormats.FileDrop))
-                e.Effect = DragDropEffects.Copy;
-            else
-                e.Effect = DragDropEffects.None;
+            e.Effect = e.Data.GetDataPresent(DataFormats.FileDrop) ? DragDropEffects.Copy : e.Effect = DragDropEffects.None;

```

Comment:
Minor refact (the second assignment is a little confusing):
```
e.Effect = e.Data.GetDataPresent(DataFormats.FileDrop) ? DragDropEffects.Copy : DragDropEffects.None;
```

However, the original with if/else might be easier to read (understand) ?